packages/compass-welcome/src/typings.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*.png' {
```

Comment:
Are we adding a .png somewhere here? (If so, I'm not sure where)

Reply:
I was going to. Just forgot to clean that up.